cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -281,4 +283,17 @@ message  AddAliasRequest {
   string access_token=1;
   Port port=2;
   string alias=3;
-}
\ No newline at end of file
+}
+
+message IsNewKurtosisVersionAvailableRequest {
+  string current_kurtosis_version = 2;
+}
+
+message IsNewKurtosisVersionAvailableResponse {
+  bool isAvailable = 1;
+  string latestVersion = 2;
+}
+
+message UpgradeKurtosisVersionRequest {
+  string api_key = 1;
```

Comment:
If this is coming from the EM UI then it should take an access token (jwt).

Reply:
@laurentluce access token added